content/300-guides/200-deployment/301-edge/450-deploy-to-cloudflare-workers.mdx

```diff
@@ -1,75 +1,44 @@
 
 </Admonition>
 
 </TopBlock>
 
 ## Prerequisites
 
-- Free [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) account
-- Free [Cloudflare Workers](https://workers.cloudflare.com/) account
-- Free [GitHub](https://github.com/) account
-- Node.js & NPM installed
+- A PostgreSQL database that is publicly accessible
+- [Cloudflare Workers](https://workers.cloudflare.com/) account
+- [Prisma Data Platform](https://console.prisma.io/) account
+- Node.js & npm installed
 - Git installed
 
-## 1. Get a MongoDB Atlas connection string
-
-After you create a MongoDB Atlas database cluster, you can get its connection string. By default, that connection string does not include a database name.
-
-When you import your project in the Prisma Data Platform at [step 6](#6-import-your-project-into-the-prisma-data-platform), you must provide a connection string that includes a database name.
-
-However, you do not need to manually create a database in MongoDB Atlas. The project creation workflow in the Prisma Data Platform will create a database with the name you define in the connection string.
-
-1. [Create a free shared cluster](https://www.mongodb.com/docs/atlas/tutorial/create-new-cluster/) in MongoDB Atlas.
-2. On the _Security Quickstart_ screen, complete the initial security configuration.
-   1. Add a new database user.
-   2. Click **Add My Current IP Address** so that you can connect to the MongoDB shared cluster from your current system.
-   3. Click **Finish and Close**.
-3. [Get the database cluster connection string](https://www.mongodb.com/docs/guides/atlas/connection-string/).
-4. Paste the connection string in a text editor and add the collection name `Log`.
-
-   The code sample below shows a connection string, that is originally provided by MongoDB Atlas, with the database name now added.
-
-   ```diff
-   # The first line is a default-style MongoDB Atlas connection string.
-   # The second line includes the inserted `Log` database name.
-   - mongodb+srv://<user>:<password>@cluster.mongodb.net/?retryWrites=true&w=majority
-   + mongodb+srv://<user>:<password>@cluster.mongodb.net/Log?retryWrites=true&w=majority
-   ```
-
-You now have a complete connection string. You will paste the connection string when you create your project in [6. Import your project into the Prisma Data Platform](#6-import-your-project-into-the-prisma-data-platform).
-
-For security reasons, do not keep the connection string in the text editor or save it in a text file.
-
-## 2. Set up your application
-
 Open your terminal and navigate to a location of your choice. Run the following commands to set up your application.
 
 ```terminal
-mkdir prisma-mongodb-cloudflare
-cd prisma-mongodb-cloudflare
+mkdir prisma-cloudflare-accelerate
+cd prisma-cloudflare-accelerate
 npm init -y
 npm install -D prisma typescript wrangler
```

Comment:
I don't think these steps are required with wrangler v3.

I ran `wrangler init` and it created a new directory with an existing application preset, i.e., with Hono, or Remix, Angular, Astro etc. that you can deploy.